src/raftstore/store/snap.rs

```diff
@@ -711,7 +713,9 @@ impl Snap {
                     false,
                     &mut |key, value| {
                         key_count += 1;
-                        size += key.len() + value.len();
+                        let l = key.len() + value.len();
+                        size += l;
+                        limiter.request(l as i64, 0);
```

Comment:
Call `request` once for each appropriate size of batch, eg. 128KB.

Reply:
Fixed.